Saule/StringExtensions.cs

```diff
@@ -23,7 +23,7 @@ public static string EnsureStartsWith(this string source, string start)
 
         public static string TrimJoin(this char separator, params string[] parts)
         {
-            return string.Join(separator.ToString(), parts.Select(p => p.Trim(separator)));
+            return string.Join(separator.ToString(), parts.Select(p => p?.Trim(separator)));
```

Comment:
Not sure, it was from https://github.com/joukevandermaas/saule/pull/136/commits/368ea27c0d21d5c4bd9050b52ac3aef460afa8ec by @yohanmishkin.